services/credentials.go

```diff
@@ -28,30 +28,30 @@ const (
 	credsRequestMaxAge = time.Duration(15) * time.Minute
 )
 
-type quota struct {
+type Quota struct {
 	// Max number of credentials that can be requested in a given time window.
-	count uint
+	Count uint `json:"count"`
 	// Time window in which the credential quota is calculated.
-	window time.Duration
+	Window time.Duration `json:"window"`
 	// Duration a credential is valid for
-	authValidityWindow time.Duration
+	AuthValidityWindow time.Duration `json:"authValidityWindow"`
```

Comment:
snek case for json fields

Reply:
Happy to change this, but  CreateCredentialResponse isn't using that convention, i.e.:
```
ExpiresAt int64  `json:"expiresAt"`
```

Thoughts?

Reply:
I suppose we should strive for consistency. Leave it as camel. It's too annoying to change the other response (we would have to update the UI).

Reply:
Thought the same. :( Thanks for the input!